.codeclimate.yml

```diff
@@ -8,6 +8,16 @@ engines:
     checks:
       overqualified-elements:
         enabled: false
+      order-alphabetical:
+        enabled: false
+      adjoining-classes:
+        enabled: false
+      fallback-colors:
+        enabled: false
+      ids:
+        enabled: false
+      regex-selectors:
```

Comment:
Regex is heavy and there probably is a better way to do that selector. I'd say that this remain enabled.